packages/button-theme/__tests__/Fieldset.test.tsx

```diff
@@ -0,0 +1,17 @@
+import { axe, toHaveNoViolations } from 'jest-axe';
+import { render, fireEvent } from '@testing-library/react';
+import React from 'react';
+import Fieldset from '../src/Fieldset';
+import '@testing-library/jest-dom/extend-expect';
+import 'regenerator-runtime/runtime';
+
+expect.extend(toHaveNoViolations);
+
+describe('Fieldset', () => {
+  it('Should have no accessibility violations', async () => {
+    const { container } = render(<Fieldset />);
+    const results = await axe(container);
+
+    expect(results).toHaveNoViolations();
+  });
```

Comment:
Can we add testing for passing the user props? There are some examples in the bcgov Input component that could be useful.